File: eternal_return/src/main/java/org/EternalReturn/ERCharacter/Character/fiora/Character_Fiora.java
package org.EternalReturn.ERCharacter.Character.fiora;


import org.EternalReturn.ERCharacter.ERCharacter;
import org.EternalReturn.ERPlayer.ERPlayer;

public class Character_Fiora extends ERCharacter {

    public boolean isActiveSkill = false;

    public Character_Fiora(ERPlayer player) {
        super(player);
        this.ActiveCooldownSeconds = 3;
        registerMonobehaviour(new Passive());
        registerMonobehaviour(new Active());
    }


    @Override
    public String getName() { return "fiora"; }

}

File: eternal_return/src/main/kotlin/org/EternalReturn/ERCharacter/character/fiora/Character_Fiora.kt
package org.EternalReturn.ERCharacter.character.fiora

import org.EternalReturn.ERCharacter.ERCharacter
import org.EternalReturn.ERPlayer.ERPlayer


class Character_Fiora(player: ERPlayer) : ERCharacter(player) {
    init {
        this.ActiveCooldownSeconds = 8
        registerMonobehaviour(Passive())
        registerMonobehaviour(Active())
    }


    override val name: String
        get() = "fiora"
}